client/src/views/Edit.vue

```diff
@@ -150,76 +151,30 @@
           return previous
         }, {})
       },
+    },
+    methods: {
+      toUpload () {
+        this.$router.push('upload')
+      },
 
-      getMenuStructure () {
+      getMenuStructure (intermediateFiles, keyNum) {
         const icon = 'fa fa-folder'
-        return Object.entries(this.intermediateFiles).reduce((previous, [period, value]) => {
-          previous.push({
-            title: period,
-            icon,
-            child: generateChildOfPeriod(value)
-          })
-          return previous
-        }, [])
-
-        function generateChildOfPeriod(yearValue) {
-          return Object.entries(yearValue).reduce((previous, [subject, subjectValue]) => {
-            previous.push({
-              title: subject,
-              icon,
-              child: generateChildOfSubject(subjectValue)
-            })
-            return previous
-          }, [])
-        }
-
-        function generateChildOfSubject(subjectValue) {
-          return Object.entries(subjectValue).reduce((previous, [toolType, toolTypeValue]) => {
-            previous.push({
-              title: toolType,
-              icon,
-              child: generateChildOfToolType(toolTypeValue)
-            })
-            return previous
-          }, [])
-        }
-
-        function generateChildOfToolType(toolTypeValue) {
-          return Object.entries(toolTypeValue).reduce((previous, [year, yearValue]) => {
-            previous.push({
-              title: year,
-              icon,
-              child: generateChildOfYear(yearValue)
-            })
-            return previous
-          }, [])
-        }
-
-        function generateChildOfYear(yearValue) {
-          return Object.entries(yearValue).reduce((previous, [contentType, contentTypeValue]) => {
-            previous.push({
-              title: contentType,
-              icon,
-              child: generateChildOfContentType(contentTypeValue)
-            })
-            return previous
-          }, [])
-        }
-
-        function generateChildOfContentType(contentTypeValue) {
-          return contentTypeValue.map(file => {
+        if (keyNum <= 1) {
+          return intermediateFiles.map(file => {
             return {
               title: file.fileName,
               icon: 'fa fa-file'
             }
           })
```

Comment:
arrow func で `()` の中に `{}` を入れると `return` を省略できるよ
```suggestion
          return intermediateFiles.map(file => ({
            title: file.fileName,
            icon: 'fa fa-file'
          }))
```